x-pack/plugins/observability/public/pages/alerts/alerts_table_t_grid.tsx

```diff
@@ -10,24 +10,24 @@ import type {
   ALERT_DURATION as ALERT_DURATION_TYPED,
   ALERT_SEVERITY_LEVEL as ALERT_SEVERITY_LEVEL_TYPED,
   ALERT_STATUS as ALERT_STATUS_TYPED,
-  ALERT_START as ALERT_START_TYPED,
   ALERT_RULE_NAME as ALERT_RULE_NAME_TYPED,
 } from '@kbn/rule-data-utils';
 import {
-  AlertConsumers as AlertConsumersNonTyped,
   ALERT_DURATION as ALERT_DURATION_NON_TYPED,
   ALERT_SEVERITY_LEVEL as ALERT_SEVERITY_LEVEL_NON_TYPED,
   ALERT_STATUS as ALERT_STATUS_NON_TYPED,
-  ALERT_START as ALERT_START_NON_TYPED,
   ALERT_RULE_NAME as ALERT_RULE_NAME_NON_TYPED,
+  TIMESTAMP,
   // @ts-expect-error
-} from '@kbn/rule-data-utils/target_node/alerts_as_data_rbac';
+} from '@kbn/rule-data-utils/target_node/technical_field_names';
+
+import { AlertConsumers as AlertConsumersNonTyped } from '@kbn/rule-data-utils/target_node/alerts_as_data_rbac';
```

Comment:
```suggestion
// @ts-expect-error
import { AlertConsumers as AlertConsumersNonTyped } from '@kbn/rule-data-utils/target_node/alerts_as_data_rbac';
```

Reply:
This will fail without the //@ts-expect-error as the types are now separated from the code transpilation folder and we are using target imports instead of importing from the root of the package